contracts/adapters/AutoRepay.sol

```diff
@@ -282,20 +294,23 @@ contract AutoRepay is BaseUniswapAdapter {
         aTokenTransferAmount,
         permitSignature
       );
-      uint256 contractBalanceBefore = IERC20(collateralATokenAddress).balanceOf(address(this));
+      // uint256 contractBalanceBefore = IERC20(collateralATokenAddress).balanceOf(address(this));
       LENDING_POOL.withdraw(
         repayParams.collateralAsset,
         repayParams.debtRepayAmount.add(premium),
         address(this)
       );
-      uint256 contractBalanceAfter = IERC20(collateralATokenAddress).balanceOf(address(this));
+      // uint256 contractBalanceAfter = IERC20(collateralATokenAddress).balanceOf(address(this));
       IERC20(collateralATokenAddress).safeTransfer(
         caller,
-        aTokenTransferAmount - (contractBalanceBefore - contractBalanceAfter)
+        IERC20(collateralATokenAddress).balanceOf(address(this))
       );
     }
   }
 
+  /**
+   * @dev decodes the params and returns them
+   */
```

Comment:
```suggestion
```
Docs for the sake of docs are pointless. If the function is simple enough that commentary doesn't help to explain it, then no need for commentary.